core/admin/requirements.txt

```diff
@@ -18,10 +18,8 @@ PyYAML
 PyOpenSSL
 Pygments
 dnspython
-bcrypt
```

Comment:
Hmm? it's still in requirements-prod.txt

Reply:
Yes. Mailu doesn't use it directly but it's used by cryptography which is used by pyOpenSSL.